src/user.rs

```diff
@@ -0,0 +1,46 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+use rustc_serialize::json;
+
+use project_types::Project;
+use Error;
+
+
+pub fn new_user<P: AsRef<Path>>(project: String, public: P, name: String) -> Result<(), Error> {
```

Comment:
1. Add a doc comment on this. Look at `init::initialize_project` for an example. Make sure to mark that its impure.
2. I found it kind of confusing how you're mapping command line params to this function's params.
  1. Is project the path to the project root? It looks like it based on how you're finding the protonfile from here. Remove this as a param, and assume that for any commands besides `init` that the project root is the current working directory.
  2. `public` -> `public_key_path`.
  3. `name`. This should be an `&str`. There's no reason for new_user to need an owned copy of the name. If you want me to explain this in more detail, let me know.